src/Components/Home.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { ring1, ring2, ring3, ring4, ring5, ring7 } from "../assets/images/index"
+
+// import "./Components/Header"
+
+function Home() {
+  return (
+    <div>
+        <div className="home">
+        <img src={ring7} />
+    </div>
+  
+      
+    <div>
+        <button className="button"><a href="/products-btn/product.html">Shop now</a></button>
+    </div>
+
+
+    <div className="small-container">
+        <div className="row">
+            <div class="col-2">
+                <div className="offer">
+                    <img src={ring1} className="offer-im" />
```

Comment:
you could create an array of all the images and loop through them, it will save the repeated code